lib/pkg-config.rb

```diff
@@ -520,7 +520,7 @@ def parse_pc
 
   def parse_requires(requires)
     return [] if requires.nil?
-    requires_without_version = requires.gsub(/[<>]?=\s*[\d.a-zA-Z_-]+\s*/, "")
+    requires_without_version = requires.gsub(/(<|>|<=|>=|=)\s*[\d.a-zA-Z_-]+\s*/, "")
```

Comment:
Could you avoid needless captures? 

```suggestion
    requires_without_version = requires.gsub(/(?:<|>|<=|>=|=)\s*[\d.a-zA-Z_-]+\s*/, "")
```